Source/Scene/Scene.js

```diff
@@ -2923,15 +2942,14 @@ define([
 
         var shouldRender = !this.requestRenderMode || this._renderRequested || cameraChanged || (this.mode === SceneMode.MORPHING);
 
-        var now = JulianDate.clone(time);
         if (!shouldRender && defined(this.maximumRenderTimeChange) && defined(this._lastRenderTime)) {
-            var difference = Math.abs(JulianDate.secondsDifference(this._lastRenderTime, now));
+            var difference = Math.abs(JulianDate.secondsDifference(this._lastRenderTime, time));
             shouldRender = shouldRender || difference >= this.maximumRenderTimeChange;
         }
 
         try {
             if (shouldRender) {
-                this._lastRenderTime = now;
+                this._lastRenderTime = JulianDate.clone(time, scratchTime);;
```

Comment:
Actually now that I see the updated changes just remove the scratch and call `JulianData.clone(time, this._lastRenderTime)`.